rdev/R/utils.R

```diff
@@ -252,9 +266,14 @@ render <- function(x,file=tempfile(fileext=".Rmd"),quiet=TRUE,...) {
       capture.output(cat(bl$TABLE,sep="\n")),
       "```",
       sep="\n",append=TRUE)
-  ret <- rmarkdown::render(file,quiet=quiet,output_format="pdf_document")
-  file.copy(ret,file.path(getwd(),basename(ret)))
-  return(ret)
+  
+  if(build) {
+    ret <- rmarkdown::render(file,quiet=quiet,output_format="pdf_document")
```

Comment:
would this be better to wrap in a tryCatch block, so you can bubble up times when the rendering fails (for whatever reason it might)?